test/__snapshots__/index.spec.ts.snap

```diff
@@ -4201,7 +4203,7 @@ import type { ModelWithDictionary } from './ModelWithDictionary';
  * This is a model with nested 'any of' property with a type null
  */
 export type CompositionWithNestedAnyAndTypeNull = {
-    propA?: ((ModelWithDictionary | null) | (ModelWithArray | null));
+    propA?: (Array<(ModelWithDictionary | null)> | Array<(ModelWithArray | null)>);
```

Comment:
@nicolas-chaulet these should be arrays https://github.com/ferdikoomen/openapi-typescript-codegen/pull/1683/files#diff-f96c002010932e16fe6778d2c9ff94a719d2b7ddc38076eabc474b618b9be108R4117